src/main/java/qna/domain/Answer.java

```diff
@@ -1,46 +1,43 @@
 package qna.domain;
 
-import org.springframework.data.annotation.CreatedDate;
-import org.springframework.data.annotation.LastModifiedBy;
-import org.springframework.data.annotation.LastModifiedDate;
-import org.springframework.data.jpa.domain.support.AuditingEntityListener;
-import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
-import qna.NotFoundException;
-import qna.UnAuthorizedException;
-
+import java.time.LocalDateTime;
+import java.util.Objects;
 import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.EntityListeners;
+import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
 import javax.persistence.Lob;
-import javax.persistence.Table;
-import java.time.LocalDateTime;
-import java.util.Objects;
+import javax.persistence.ManyToOne;
+import org.springframework.data.annotation.LastModifiedDate;
+import qna.NotFoundException;
+import qna.UnAuthorizedException;
 
-@EntityListeners(AuditingEntityListener.class)
 @Entity
-public class Answer {
+public class Answer extends BaseEntity {
+
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Id
-    private Long id;
+    protected Long id;
 
     @Lob
     private String contents;
 
-    @CreatedDate
     @Column(nullable = false)
-    private LocalDateTime createdAt;
-
     private boolean deleted = false;
 
-    private Long questionId;
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "question_id")
```

Comment:
```suggestion
    @JoinColumn
```

name이 없다면 어떻게 동작하는지 알고 계시나요?

Reply:
default값은 연관관계 대상인 엔티티의 이름 + "_" + 키컬럼 이름으로 형성된다고 나와있네요.
저희가 원하는 (설정한) 이름과 같게 되는데 이런 경우 명시해줄 필요가 없나요?

Reply:
팀 내에 규칙을 정하시면 될 것 같습니다 ㅎㅎ 다른 곳은 전부 생략하는 상태에서 해당 부분만 작성해준다면 그 이유가 명확해야 할 것 같아요!